src/python/pants/backend/awslambda/python/target_types.py

```diff
@@ -105,21 +107,19 @@ class PythonAwsLambdaRuntime(PythonFaaSRuntimeField):
         """
     )
 
-    # https://gallery.ecr.aws/lambda/python
-    known_runtimes_docker_repo = "public.ecr.aws/lambda/python"
```

Comment:
Not sure if this field should be deprecated (so that plugins can retain access to it) or simply removed as I did here.

Reply:
Just removing is fine, I think. Theoretically a breaking change, but I'd be surprised if it was one in practice!